src/core/caniuse.js

```diff
@@ -35,7 +36,7 @@ const BROWSERS = new Map([ // browser name dictionary
 
 export const name = "core/caniuse";
 
-export async function run(conf, doc, cb) {
+export function run(conf, doc, cb) {
```

Comment:
as this is now sync again, we can just:

```
export function run(conf) {
```

And remove `cb();`

Apologies for getting you to change this so many times. I think we got it this time tho 🤞